diff --git a/node_modules/@hashicorp/react-docs-page/index.tsx b/node_modules/@hashicorp/react-docs-page/index.tsx
index c804b63..bae4571 100644
--- a/node_modules/@hashicorp/react-docs-page/index.tsx
+++ b/node_modules/@hashicorp/react-docs-page/index.tsx
@@ -11,7 +11,7 @@ import Content from '@hashicorp/react-content'
 import DocsSidenav from '@hashicorp/react-docs-sidenav'
 import { NavData } from '@hashicorp/react-docs-sidenav/types'
 import HashiHead from '@hashicorp/react-head'
-import { MDXRemote, MDXRemoteSerializeResult } from 'next-mdx-remote'
+import { MDXRemote, MDXRemoteSerializeResult } from 'next-mdx-remote-v1'
 import { SearchProvider } from '@hashicorp/react-search'
 import { AlgoliaConfigObject } from '@hashicorp/react-search/types'
 import type { Products } from '@hashicorp/platform-product-meta'
diff --git a/node_modules/@hashicorp/react-docs-page/render-page-mdx.ts b/node_modules/@hashicorp/react-docs-page/render-page-mdx.ts
index 81d9937..6fc1c2a 100644
--- a/node_modules/@hashicorp/react-docs-page/render-page-mdx.ts
+++ b/node_modules/@hashicorp/react-docs-page/render-page-mdx.ts
@@ -3,8 +3,8 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-import type { MDXRemoteSerializeResult } from 'next-mdx-remote'
-import { serialize } from 'next-mdx-remote/serialize'
+import type { MDXRemoteSerializeResult } from 'next-mdx-remote-v1'
+import { serialize } from 'next-mdx-remote-v1/serialize'
 import { Pluggable } from 'unified'
 import grayMatter from 'gray-matter'
 interface Options {
